### PR TITLE
[RLlib] Bandit documentation enhancements.

### PR DESCRIPTION
**Pull request by sven1977**
Bandit documentation enhancements.

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: doc/source/rllib/rllib-algorithms.rst [hunk 1]
```diff
@@ -671,12 +673,14 @@ Tuned examples: `CartPole-v0 <https://github.com/ray-project/ray/blob/master/rll
...
 
+.. _bandits:
+
 Contextual Bandits
 ~~~~~~~~~~~~~~~~~~
 
 The Multi-armed bandit (MAB) problem provides a simplified RL setting that
-involves learning to act under one situation only, i.e. the state is fixed.
-Contextual bandit is extension of the MAB problem, where at each
+involves learning to act under one situation only, i.e. the observation/state is fixed.
```
Comment (gjoliver) [inline doc/source/rllib/rllib-algorithms.rst:682]: from this paper (http://rob.schapire.net/papers/www10.pdf) MAB is a special case of contextual bandit where the context (user) and the arms are both fixed.
maybe we can say "i.e., the context and arms are both fixed."

**Review by gjoliver — APPROVED** [review]
thanks for the update. just a minor suggestion.

File: doc/source/rllib/rllib-algorithms.rst (hunk 1 above)
Comment (sven1977) [reply doc/source/rllib/rllib-algorithms.rst:682]: Perfect, thanks for the hint and the review @gjoliver ! Will add this before merging.

Comment (sven1977) [inline doc/source/rllib/rllib-algorithms.rst:682]:
```suggestion
involves learning to act under one situation only, i.e. the context (observation/state) and arms (actions/items-to-select) are both fixed.
```

**Pull request merged**